examples/sampling/node_classification.py

```diff
@@ -0,0 +1,240 @@
+"""
+If you want a deeper understanding of node classification. You can
+read the example in the  ``examples/pytorch/graphsage/node_classification.py``
+"""
+
+import argparse
+
+import dgl
+import dgl.nn as dglnn
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torchmetrics.functional as MF
```

Comment:
Consider including a README file to specify the version of the dependencies like `torchmetrics`.